modules/objectdb_profiler/editor/data_viewers/summary_view.cpp

```diff
@@ -0,0 +1,277 @@
+	VBoxContainer *explainer_lines = memnew(VBoxContainer);
+	explainer_text->add_child(explainer_lines);
+	Label *l1 = memnew(Label(TTR("Press 'Take ObjectDB Snapshot' to snapshot the ObjectDB.")));
+	Label *l2 = memnew(Label(TTR("Memory in Godot is either owned natively by the engine or owned by the ObjectDB.")));
+	Label *l3 = memnew(Label(TTR("ObjectDB Snapshots capture only memory owned by the ObjectDB.")));
+	l1->set_horizontal_alignment(HorizontalAlignment::HORIZONTAL_ALIGNMENT_CENTER);
+	l2->set_horizontal_alignment(HorizontalAlignment::HORIZONTAL_ALIGNMENT_CENTER);
+	l3->set_horizontal_alignment(HorizontalAlignment::HORIZONTAL_ALIGNMENT_CENTER);
+	explainer_lines->add_child(l1);
+	explainer_lines->add_child(l2);
+	explainer_lines->add_child(l3);
+
+	ScrollContainer *sc = memnew(ScrollContainer);
+	sc->set_anchors_preset(LayoutPreset::PRESET_FULL_RECT);
+	sc->set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	sc->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	content->add_child(sc);
+
+	blurb_list = memnew(VBoxContainer);
+	sc->add_child(blurb_list);
+	blurb_list->set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	blurb_list->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+}
+
+void SnapshotSummaryView::show_snapshot(GameStateSnapshot *p_data, GameStateSnapshot *p_diff_data) {
+	SnapshotView::show_snapshot(p_data, p_diff_data);
+	explainer_text->set_visible(false);
+
+	String snapshot_a_name = diff_data == nullptr ? TTR("Snapshot") : TTR("Snapshot A");
+	String snapshot_b_name = TTR("Snapshot B");
+
+	_push_overview_blurb(snapshot_a_name + " " + TTR("Overview"), snapshot_data);
+	if (diff_data) {
+		_push_overview_blurb(snapshot_b_name + " " + TTR("Overview"), diff_data);
+	}
+
+	_push_node_blurb(snapshot_a_name + " " + TTR("Nodes"), snapshot_data);
+	if (diff_data) {
+		_push_node_blurb(snapshot_b_name + " " + TTR("Nodes"), diff_data);
+	}
+
+	_push_refcounted_blurb(snapshot_a_name + " " + TTR("RefCounteds"), snapshot_data);
+	if (diff_data) {
+		_push_refcounted_blurb(snapshot_b_name + " " + TTR("RefCounteds"), diff_data);
+	}
+
+	_push_object_blurb(snapshot_a_name + " " + TTR("Objects"), snapshot_data);
+	if (diff_data) {
+		_push_object_blurb(snapshot_b_name + " " + TTR("Objects"), diff_data);
+	}
+}
+
+void SnapshotSummaryView::clear_snapshot() {
+	// Just clear out the blurbs and leave the explainer.
+	for (int i = 0; i < blurb_list->get_child_count(); i++) {
+		blurb_list->get_child(i)->queue_free();
+	}
+	snapshot_data = nullptr;
+	diff_data = nullptr;
+	explainer_text->set_visible(true);
+}
+
+SummaryBlurb::SummaryBlurb(const String &p_title, const String &p_rtl_content) {
+	add_theme_constant_override("margin_left", 2);
+	add_theme_constant_override("margin_right", 2);
+	add_theme_constant_override("margin_top", 2);
+	add_theme_constant_override("margin_bottom", 2);
+
+	label = memnew(RichTextLabel);
+	label->add_theme_constant_override(SceneStringName(line_separation), 6);
+	label->set_fit_content(true);
```

Comment:
```suggestion
	label->set_text_direction(Control::TEXT_DIRECTION_INHERITED);
	label->set_fit_content(true);
```

`RichTextLabel` defaults to `AUTO` (each paragraph direction determined from content), with a mix of translatable and non-translatable messages in can be a mess in right-to-left, so probably should be forced to follow layout direction.